.github/workflows/pr-labeler.yaml

```
@@ -0,0 +1,16 @@
+name: pr-labeler
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@v2
```

v4 is the latest?
https://github.com/actions/labeler/releases/tag/v4

```suggestion
      - uses: actions/labeler@v4
```

---

Reflected by https://github.com/autowarefoundation/autoware.universe/pull/2266/commits/1a13aa2e91c91c50793285fa943cda75fd678784